priv/repo/migrations/20170213162440_add_post_belongs_to_user.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Healthlocker.Repo.Migrations.AddPostBelongsToUser do
+  use Ecto.Migration
+
+  def change do
+    alter table(:posts) do
+      add :user_id, :integer
```

Comment:
I think we should declare this as a [`reference`](https://hexdocs.pm/ecto/Ecto.Migration.html#references/2) as this will then create the associated foreign key for us.